src/main/java/no/sikt/nva/scrapers/DublinCoreScraper.java

```diff
@@ -47,8 +49,13 @@ private DublinCoreScraper() {
 
     }
 
-    public static Record validateAndParseDublinCore(DublinCore dublinCore, BrageLocation brageLocation) {
-        var errors = DublinCoreValidator.getDublinCoreErrors(dublinCore, brageLocation);
+    public static Record validateAndParseDublinCore(DublinCore dublinCore, BrageLocation brageLocation,
+                                                    boolean enableOnlineValidation) {
+        var errors = new ArrayList<ErrorDetails>();
+        if (enableOnlineValidation) {
+            errors.addAll(DublinCoreOnlineValidator.getDublinCoreErrors(dublinCore));
+        }
+        errors.addAll(DublinCoreValidator.getDublinCoreErrors(dublinCore, brageLocation));
```

Comment:
Dette er merkelig.

Metode-navnet inni if-blokka indikerer 56 + 68 indikerer at hvis enableOnlineValidation = tru så får vi to identiske errorlister. Rart

Reply:
den blir logget two forskjellige errorer, den ene fra online validatoren, og den andre fra offline. Men doien vil være den samme ja